src/legacy/core_plugins/data/public/expressions/expressions_service.ts

```diff
@@ -46,6 +46,7 @@ export interface Result {
   type: string;
   as?: string;
   value?: unknown;
+  error?: unknown;
```

Comment:
I use unknown for stuff I'm not sure about the shape. It is an object in practice but as these types are only temporary anyway I didn't bother defining it because I don't rely on it anywhere.